### PR TITLE
W-11365581 Exchange enable and disable public portal jh

### PR DESCRIPTION
**Pull request by JennyHajee**
# Writer's Quality Checklist

Before merging your PR, did you:

- [x] Run spell checker
- [x] Run link checker to check for broken xrefs
- [x] Check for orphan files
- [x] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [x] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
+= Enable and Disable a Public Portal
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+All organizational public portals in Exchange created after July 2022 are disabled by default. You cannot sign in to view assets or share new assets on your public portal unless you enable them. You must be an Exchange administrator to enable or disable a public portal.
```
Comment (hiuxmaycry) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]: The word "organizational" is a bit strange considering that there is no "non" organizational public portals 🤔. I know this was originally written by me 😛  but maybe we can delete that word and leave "All public portals in Exchange ...". What do you think?

Comment (JennyHajee) [reply modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]: Yes, that makes sense to me. I will change "organizational' to "all".

**Review by hiuxmaycry — DISMISSED** [review]

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
+= Enable and Disable a Public Portal
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+All public portals in Exchange created after July 2022 are disabled by default. You cannot sign in to view assets or share new assets on your public portal unless you enable them. You must be an Exchange administrator to enable or disable a public portal.
+
+To determine of your public portal is enabled or disabled, use the following request:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
To determine if your public portal is enabled or disabled, use the following request:
```

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+----
+curl "https://anypoint.mulesoft.com/exchange/api/v2/portals/YOUR_ORGANIZATION_DOMAIN/status"
+----
+
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc:14]:
```suggestion
```

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+== Enable a Public Portal
+
+All assets that are shared in the public portal do not require authorization to view the details of an asset or download the specifications.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
You do not need authorization to view the details or download the specifications of assets shared in the public portal.
```

Comment (IsaacEldridge) [reply modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]: the sentence was just a little wonky toward the end so trying to revise it, maybe could also write

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+To start using your organization's public portal and sharing assets publicly, you must enable the portal.
+
+. Get an access token from an Anypoint Platform user account that has the Organization Administrator permission in the root organization:
++
+[source,console]
+----
+ACCESS_TOKEN=$(curl https://anypoint.mulesoft.com/accounts/login -XPOST -d "username=YOUR_USERNAME&password=YOUR_PASSWORD" | jq -r ".access_token")
+----
++
+. Use the access token to change the status of your organization's public portal to enabled status:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc:26]:
```suggestion
. Use the access token to change the status of your organization's public portal to enabled:
```

Comment (IsaacEldridge) [reply modules/ROOT/pages/enable-and-disable-a-public-portal.adoc:26]: Or "Use the access token to enable organization's public portal:

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
++
+----
+curl -X PUT https://anypoint.mulesoft.com/exchange/api/v2/portals/YOUR_ORGANIZATION_DOMAIN/status -d "enabled" -H "content-type: text/plain" -H "authorization: bearer $ACCESS_TOKEN"
+----
+
+
+== Disable a Public Portal
+
+You can disable your public portal if you do not want to have any publicly shared information about your assets.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
If you do not want to publicly share any information about your assets, disable your public portal. 
```

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+----
+curl -X PUT https://anypoint.mulesoft.com/exchange/api/v2/portals/YOUR_ORGANIZATION_DOMAIN/status -d "enabled" -H "content-type: text/plain" -H "authorization: bearer $ACCESS_TOKEN"
+----
+
+
+== Disable a Public Portal
+
+You can disable your public portal if you do not want to have any publicly shared information about your assets. 
+
+To disable your organization's public portal, all the assets that are shared in the public portal must be set to private.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
To disable your organization's public portal, set all the assets that are shared in the public portal to private. 
```

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+----
+
+
+== Disable a Public Portal
+
+You can disable your public portal if you do not want to have any publicly shared information about your assets. 
+
+To disable your organization's public portal, all the assets that are shared in the public portal must be set to private. 
+
+. To view which assets are shared in the public portal, use the following request:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
To view which assets are shared in the public portal, use the following request:
```

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
++
+[source,console]
+----
+curl 'https://anypoint.mulesoft.com/exchange/api/v2/portals/YOUR_ORGANIZATION_DOMAIN/assets' | jq '.[] | [.groupId,.assetId]'
+----
+
+. Perform the following steps to set all the assets that are shared publicly in the public portal to private:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
Perform the following steps to set all the assets that are shared publicly in the public portal to private:
```

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+.. Select the asset in Exchange and click *Share*.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
. Select the asset in Exchange and click *Share*.
```

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+.. Select *Public - Publish in the on the public portal*.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
. Select *Public - Publish in the on the public portal*.
```

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+.. Unselect the version checkbox.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
. Unselect the version checkbox.
```

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
++
+The version of the asset is set to *Private*.
++
+.. Select *Save*.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
. Select *Save*.
```

File: modules/ROOT/pages/to-share-api-asset-to-portal.adoc [hunk 14]
```diff
@@ -35,13 +35,18 @@ If no action occurs, refresh your browser, log in again, and add the asset again
...
 Follow these steps to remove an asset from a portal and make the asset private.
 
-Removing an asset from a portal does not delete the asset. It is still available internally.
+Removing an asset from a portal does not delete the asset. The asset is still available internally.
 
-. Select the asset in Exchange and select *Share* -> *Add to the Public Portal*.
-. Uncheck the version field checkbox.
+. Select the asset in Exchange and click *Share*.
+. Select *Public - Publish in the on the public portal*.
+. Unselect the version checkbox.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-share-api-asset-to-portal.adoc]:
```suggestion
. Deselect the version checkbox.
```

Comment (IsaacEldridge) [reply modules/ROOT/pages/to-share-api-asset-to-portal.adoc]: This is in the style guide, which I think is really weird. I usually write "Clear the checkbox" 

who knows... 🤷

File: modules/ROOT/pages/to-share-api-asset-to-portal.adoc [hunk 15]
```diff
@@ -35,13 +35,18 @@ If no action occurs, refresh your browser, log in again, and add the asset again
...
++
+The version of the asset is set to *Private*.
++
 . Select *Save*.
 . In Exchange, refresh the browser screen.
-. In the asset's *Overview* the *Visibility* setting is now *Private*.
++
+In the asset's *Overview* the *Visibility* setting is now *Private*.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-share-api-asset-to-portal.adoc]:
```suggestion
In the asset's *Overview*, the *Visibility* setting is now *Private*.
```

**Review by IsaacEldridge — DISMISSED** [review]

File: modules/ROOT/pages/to-share-api-asset-to-portal.adoc (hunk 14 above)
Comment (JennyHajee) [reply modules/ROOT/pages/to-share-api-asset-to-portal.adoc]: I agree with you. I like "clear the checkbox".

**Review by IsaacEldridge — DISMISSED** [review]

**Review by IsaacEldridge — DISMISSED** [review]

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc [hunk 16]
```diff
@@ -0,0 +1,68 @@
+= Enable and Disable a Public Portal
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+All public portals in Exchange created after July 2022 are disabled by default. You cannot sign in to view assets or share new assets on your public portal unless you enable them. You must be an Exchange administrator to enable or disable a public portal.
```
Comment (hiuxmaycry) [inline modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]:
```suggestion
All public portals in Exchange are enabled by default. If disable your public portal, you cannot sign in to view assets or share new assets on your public portal unless you re-enable them. You must be an Exchange administrator to enable or disable a public portal.
```

Comment (hiuxmaycry) [reply modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]: After a QUAD meeting this was decided:

Comment (hiuxmaycry) [reply modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]: Should we plan for Phase II of "Ability to disable public portal"?
Jul 12 Quad meeting: We'll plan Phase II for Q3 + keep public portals enabled by default until Phase II is completed

Comment (hiuxmaycry) [reply modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]: So we should change this text

**Review by hiuxmaycry — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc (hunk 16 above)
Comment (JennyHajee) [reply modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]: Thanks for letting me know. I will update the content.

**Review by bjoson-mulesoft — APPROVED** [review]

File: modules/ROOT/pages/enable-and-disable-a-public-portal.adoc (hunk 16 above)
Comment (JennyHajee) [reply modules/ROOT/pages/enable-and-disable-a-public-portal.adoc]: updated

**Review by hiuxmaycry — APPROVED** [review]
LGTM

**Pull request merged**